### DISCUSSION TITLE
2024/04/03/%E3%80%8A%E5%B4%A9%E5%9D%8F3%E3%80%8B%E5%86%8D%E4%B8%80%E6%AC%A1%E7%AB%99%E5%88%B0%E4%BA%86%E5%8D%81%E5%AD%97%E8%B7%AF%E5%8F%A3/

### DISCUSSION THREAD

**Discussion by giscus** [bingxin666/blogmessages#4] [Announcements]
# 2024/04/03/%E3%80%8A%E5%B4%A9%E5%9D%8F3%E3%80%8B%E5%86%8D%E4%B8%80%E6%AC%A1%E7%AB%99%E5%88%B0%E4%BA%86%E5%8D%81%E5%AD%97%E8%B7%AF%E5%8F%A3/

米哈游，成也《崩坏3》，败也《崩坏3》

https://blog.byteloid.one/2024/04/03/%E3%80%8A%E5%B4%A9%E5%9D%8F3%E3%80%8B%E5%86%8D%E4%B8%80%E6%AC%A1%E7%AB%99%E5%88%B0%E4%BA%86%E5%8D%81%E5%AD%97%E8%B7%AF%E5%8F%A3/

**Comment by HaoTian22** [comment]
崩3开服元老玩家，打到了主线11章，但是后来因为手机内存不够玩，iPad登录不了应用宝服，”被迫“退坑，感觉已经不太熟了
